.github/mergify.yml

```diff
@@ -0,0 +1,17 @@
+pull_request_rules:
+  - name: automatic merge on CI success
+    conditions:
+      - base=master
+      - check-success=ci/hercules/evaluation
```

Comment:
If we ever have github actions than there are cases where github token is required to merge automatic pull requests. I think this is when an automatic PR is created and mergify does not need to merge any updates into it because it's up-to-date. In those cases github's CI is not triggered.